.flake8

```diff
@@ -1,12 +1,14 @@
 [flake8]
-exclude=
-    .tox
```

Comment:
Removing this seems to have made the fs scan step unforgivably slow.

Using `extend-exclude` should fix this: https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-extend-exclude.

Reply:
Let's consider that as part of jaraco/skeleton#28 or possibly as a separate issue.

Reply:
Yeah, I posted the comments here because this diff has more context in-place.